src/uct/cuda/cuda_copy/cuda_copy_md.c

```diff
@@ -50,6 +56,14 @@ static ucs_config_field_t uct_cuda_copy_md_config_table[] = {
      " is registered. Otherwise only the user specified region is registered.",
      ucs_offsetof(uct_cuda_copy_md_config_t, max_reg_ratio), UCS_CONFIG_TYPE_DOUBLE},
 
+    {"PREF_LOC", "cpu",
+     "System device designation of a CUDA managed memory buffer"
+     " whose preferred location attribute is not set.\n"
+     " cpu - sys_device is set to UCS_SYS_DEVICE_UNKNOWN.\n"
```

Comment:
cpu - Assume the buffer is on the CPU
gpu - Assume the buffer is on the GPU corresponding to buffer's GPU context

the user facing documentation cannot use terminology from UCX code such as variables/constant names, it should document the behavior